### DISCUSSION TITLE
Does this support rebinding controls?

### DISCUSSION THREAD

**Discussion by joelgb2194** [Orange-Panda/Rewired-Glyphs#1] [Q&A] [answered]
Does your extension support the case where players will change their input bindings through the rewired control mapper? 

We have the issue where we can't display inline sprites inside TextMeshPro text components because we have no way of updating the sprite reference if the player has rebound their inputs for that sprite

**Comment by Orange-Panda** [comment, marked as answer]
This feature should work but will require informing the glyph system that controls have changed.

The system only updates glyphs when they are considered dirty. By default this occurs when a players hardware device is switched. 

You can manually inform the system that the glyphs are out of date by invoking `InputGlyphs.MarkGlyphsDirty`. The RewiredGlyphManger will then update the components in the next frame.
